src/App.js

```diff
@@ -1,20 +1,60 @@
 import './App.css';
-import CoachInfo from './CoachInfo';
-import ExerciseSearch from './ExerciseSearch';
+//import CoachInfo from './CoachInfo';
+//import ExerciseSearch from './ExerciseSearch';
 import Header from './Header';
```

Comment:
its here header is defined but never used is the error. this may or may not fix the issue that just the current error